app/src/main/java/com/example/parkingsystem/mvp/model/ReservationModel.java

```diff
@@ -0,0 +1,31 @@
+package com.example.parkingsystem.mvp.model;
+
+import android.widget.DatePicker;
+import android.widget.TimePicker;
+import com.example.parkingsystem.mvp.contract.ReservationContract;
+
+public class ReservationModel implements ReservationContract.ReservationModelContract {
+
+    private DatePicker datePicker;
```

Comment:
The DatePicker and the TimePicker are android views.. this should be in the view, not in the model

Reply:
Done